Dockerfile

```diff
@@ -11,6 +11,9 @@ COPY transform/gdc/*.py /opt/
 # Ensembl Gene / Transcript / Exon
 COPY transform/ensembl/*.py /opt/
 
+# Copy RNA Transform code
+COPY transform/rna-matrix/transform-rna-matrix.py /opt/
```

Comment:
I'm concerned that the repo layout is changing inside the docker container. The layout of the tree should stay the same.